### PR TITLE
feat(listResources_endpoint): add controller and spec for listResources

### PR DESCRIPTION
**Pull request by jjsantos01**
Add method `resources_of_learning_unit` to `api_resource_controller` that returns an array of resources objects belonging to a learning unit. Also, modify routes and spec to test this endpoint.

### CODE REVIEW

File: spec/integration/api_resources_controller_spec.rb [hunk 1]
```diff
@@ -38,4 +38,63 @@
...
+        schema type: :array, items: {
+          properties: {
+            id: { type: :integer },
+            name: { type: :string },
+            url: { type: :string },
+            average_evaluation: { type: :string }
+          }
+        }
+        let(:learning_unit) { create(:learning_unit) }
+        let(:learning_unit_id) { learning_unit.id }
```
Comment (rtichauerv) [inline spec/integration/api_resources_controller_spec.rb:59]: Not that important, but considering that you are only using the learning_unit_id variable, you could reduce this to a one-liner `let(:learning_unit_id) { create(:learning_unit).id }`

**Review by rtichauerv — APPROVED** [review]
Perfect, very clean PR 💯

File: spec/integration/api_resources_controller_spec.rb (hunk 1 above)
Comment (jjsantos01) [reply spec/integration/api_resources_controller_spec.rb:59]: I use the `:learning_unit` to create a new resource (line 77)

Comment (jjsantos01) [reply spec/integration/api_resources_controller_spec.rb]: Done!

**Pull request merged**